### PR TITLE
Improving READ.md

### PR DESCRIPTION
**Pull request by aman5062**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 This repository contains the source code for the [cantera.org](https://cantera.org) website. The
 site is built using the [Nikola](https://getnikola.com) static site generator.
 
-## To build the website
+## How to build the website?
```
Comment (decaluwe) [inline README.md:6]: I would remove the question mark, here.

Comment (decaluwe) [inline README.md:6]:
```suggestion
## How to build the website:
```

File: README.md [hunk 2]
```diff
@@ -15,14 +15,14 @@ site is built using the [Nikola](https://getnikola.com) static site generator.
 8. Inside the website repo, install the required packages: `pip install -r requirements.txt`
 9. Build the website and open in browser: `nikola auto -b`
 
-## To add a language of examples
+## How to add a language of examples?
```
Comment (decaluwe) [inline README.md:18]:
```suggestion
## How to add a language of examples:
```

File: README.md [hunk 3]
```diff
@@ -15,14 +15,14 @@ site is built using the [Nikola](https://getnikola.com) static site generator.
...
 
 1. Copy one of the existing `render_*_examples.py` and `render_*_examples.plugin`
 2. Add the source folder as the key and destination folder to the `EXAMPLES_FOLDERS` dictionary in `conf.py`
 3. Edit the `render_*_examples.py` file to build the examples
 4. See [this blog post](https://bryanwweber.com/writing/personal/2018/12/22/writing-task-plugins-for-nikola/) for more information about building Nikola tasks
 
-## To add an example category for Jupyter or Python
+## How to add an example category for Jupyter or Python?
```
Comment (decaluwe) [inline README.md:25]:
```suggestion
## How to add an example category for Jupyter or Python:
```

File: README.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ site is built using the [Nikola](https://getnikola.com) static site generator.
    3. `summaries`: A empty dictionary that gets filled with keys that are the example filename and values that are the summary from that example
 4. Add a new key to that dictionary with the folder name as the key and fill in the `name` key in the nested dictionary and set the `files` and `summaries` keys to the empty list and empty dictionary, respectively
 
-## To add a version's release notes
+## How to add a version's release notes?
```
Comment (decaluwe) [inline README.md:35]:
```suggestion
## How to add a version's release notes:
```

**Review by decaluwe — CHANGES_REQUESTED** [review]
Thanks, @aman5062.  I like the addition of "How" (as in "How to"), but would replace the question mark with a colon, in all cases.

Thanks again!

**Comment by bryanwweber** [comment]
@aman5062 I'm going to close this since you didn't respond to the comments after a few weeks. Thanks for your help, if you want to modify the suggestions that @decaluwe mentioned, then please feel free to reopen the pull request.

**Comment by aman5062** [comment]
Thanks

**Pull request closed without merging**